### PR TITLE
defer should not be in loop

### PR DESCRIPTION
**Pull request by soniasingla**
Signed-off-by: Sonia Singla <soniasingla.1812@gmail.com>

### CODE REVIEW

**Comment by akhilerm** [comment]
@Z0Marlin Can you review this PR.?

**Review by akhilerm — COMMENTED** [review]
@soniasingla Can you please check on the PR title and change introduced?

cc: @Z0Marlin PTAL.

File: cmd/ndm_daemonset/probe/mountprobe_test.go [hunk 1]
```diff
@@ -67,7 +67,9 @@ func TestMountProbeFillBlockDeviceDetails(t *testing.T) {
 		if err != nil {
 			t.Fatal(err)
 		}
-		defer exitFakeRoot()
+		defer func() {
```
Comment (Z0Marlin) [inline cmd/ndm_daemonset/probe/mountprobe_test.go:70]: I'm not sure why this change is needed. Can you please explain the change here

**Review by Z0Marlin — COMMENTED** [review]
@soniasingla have left a comment on the changes made. Also, I'm confused about the PR title. It does not seem to summarize the changes. Could you please take a look at it. Thanks :)

**Comment by akhilerm** [comment]
@soniasingla Can you update on this PR?

**Comment by akhilerm** [comment]
cc: @soniasingla
